docker-compose.yml

```diff
@@ -331,10 +331,9 @@ services:
       - RECOMMENDATION_SERVICE_PORT
       - PRODUCT_CATALOG_SERVICE_ADDR
       - OTEL_PYTHON_LOG_CORRELATION=true
-      - OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
-      # OTEL_EXPORTER_OTLP_METRICS_ENDPOINT # Not working for Python OTLP exporter
+      - OTEL_METRICS_EXPORTER=console,otlp
```

Comment:
Using `console` seems unnecessary, it just fills up the log.

Reply:
Removed. :)